### PR TITLE
link MCS2 PVs to the smaract class

### PR DESCRIPTION
**Pull request by vespos**
## Description
Link MCS2 PVs to the SmarAct class.

## Motivation and Context
SmarAct IOCs are defined either with MSC or MSC2 PVs base.

## How Has This Been Tested?
xpp3 with a MSC2 PV defined in the questionnaire

## Where Has This Been Documented?
N/A

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- espov
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst:30]:
```suggestion
- vespos
```
gets me every time too, these should be github usernames

File: docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
+1161 MSC2 PV returning SmarAct device class
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- _GetMotorClass: Add MSC2 PV to motor_types and link it to SmarAct device class
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst:10]:
```suggestion
- _GetMotorClass: Add MCS2 PV to motor_types and link it to SmarAct device class
```

File: docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
+1161 MSC2 PV returning SmarAct device class
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst:1]:
```suggestion
1161 MCS2 PV returning SmarAct device class
```

**Review by tangkong — APPROVED** [review]
LGTM, a couple MSC/MCS swaps in the pre-release notes but otherwise good to go

File: docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
+1161 MCS2 PV returning SmarAct device class
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- _GetMotorClass: Add MCS2 PV to motor_types and link it to SmarAct device class
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst:10]: nitpick: I think referring to private methods here is not correct for release notes, we should instead focus on the relevant user context, e.g. the result of this fix: that the `Motor` function will properly discover MCS2 SmarAct devices.

Comment (ZLLentz) [reply docs/source/upcoming_release_notes/1161-MSC2_PV_returning_SmarAct_device_class.rst:10]: I also might consider this a bugfix but it's ambiguous enough that I'm not sure

**Pull request merged**